lib/FileModel.php

```diff
@@ -9,13 +9,16 @@ class FileModel {
 
     /**
      * @var array $fileMimeTypesInfo
+     * @var array $sortedFileSizes
      */
     private $fileMimeTypesInfo;
+    private $fileSortingAsSize;
     /**
      * FileModel constructor.
      */
     public function __construct(){
         $this->determineFileMimeTypesInfo(\OC::$server->getUserFolder());
+        $this->determineBiggestFiles(\OC::$server->getUserFolder());

```

Comment:
We are iterating same folder 2 times. We should determine these together in terms of efficiency.

Reply:
I am trying to merge determineFileMimeTypesInfo and determineBiggestFiles functions right now. I wrote a function but it returns empty array so I am working on that right now